tempto-core/src/main/java/com/teradata/tempto/configuration/Configuration.java

```diff
@@ -75,4 +76,6 @@
     Configuration getSubconfiguration(String keyPrefix);
 
     Map<String, Object> asMap();
+
+    List<String> getStringList(String key);
```

Comment:
Move up to section with the rest of get* methods.
Consider adding getStringListMandatory methods for completeness.